packages/design-system/src/styles/utilities/_font-weight.scss

```diff
@@ -3,7 +3,7 @@
 }
 
 .ds-u-font-weight--semibold {
-  font-weight: var(--font-weight-semibold) !important;
+  font-weight: 600 !important;
```

Comment:
This is a good change, but I'd take it a step further and obliterate it from our design system. Mgov is the only theme defining a semibold and in the context of web development this has no meaning (we have `<p>` tags for normal weights, `<strong>` for bold; what would we use to indicate semibold? Is semibold distinct enough to differentiate between bold and normal weights?)

Worth considering at least, then we can delete the font files from our repo 😈

Reply:
I think you're right. Let's rip off the band-aid.